### PR TITLE
Added HashMap to handle connection objects

### PR DESCRIPTION
**Pull request by Sagar-Patil-C94**

### CODE REVIEW

File: src/main/java/com/ivoyant/DatabaseOperations.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+    // Insert Data
+    public void insertData() {
+        System.out.print("Enter Table Name : ");
+        String tableName = scanner.nextLine();
+        System.out.print("Enter Column Names (Separated By ,): ");
+        String columns = scanner.nextLine();
+        System.out.print("Enter Values List (Separated By ,): ");
+        String values = scanner.nextLine();
+        int count = DataOperations.insertData(connectionMap.get(key), tableName, columns, values);
```
Comment (shreeharibv) [inline src/main/java/com/ivoyant/DatabaseOperations.java:154]: How many rows can we insert at a time here? What if we need to insert 3-4 rows at a time?

File: src/main/java/com/ivoyant/DatabaseOperations.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        if (count == -1) {
+            System.out.println("Insertion UnSuccessful");
+        } else {
+            System.out.println("Inserted " + count + " Rows Successfully.");
+        }
+    }
+
+    // Display Data
+    public void displayData() {
+        System.out.print("Enter Column List : ");
```
Comment (shreeharibv) [inline src/main/java/com/ivoyant/DatabaseOperations.java:164]: Does it work If I give column list as '*'?

**Review by shreeharibv — CHANGES_REQUESTED** [review]
Also @Sagar-Patil-C94, try to add a case where I can manually query. Create a method for having custom queries in the plugin and give the query from the child.

File: src/main/java/com/ivoyant/DatabaseOperations.java (hunk 1 above)
Comment (Sagar-Patil-C94) [reply src/main/java/com/ivoyant/DatabaseOperations.java:154]: We can insert as many we want like this
('sagar','java developer',2000),('virat','python developer',3000).....

File: src/main/java/com/ivoyant/DatabaseOperations.java (hunk 2 above)
Comment (Sagar-Patil-C94) [reply src/main/java/com/ivoyant/DatabaseOperations.java:164]: Yes it will work

**Pull request closed without merging**